com.unity.render-pipelines.universal/Editor/2D/Light2DEditor.cs

```diff
@@ -194,7 +194,7 @@ void OnEnable()
             m_ShapeLightParametricSides = serializedObject.FindProperty("m_ShapeLightParametricSides");
             m_ShapeLightParametricAngleOffset = serializedObject.FindProperty("m_ShapeLightParametricAngleOffset");
             m_ShapeLightFalloffOffset = serializedObject.FindProperty("m_ShapeLightFalloffOffset");
-            m_ShapeLightSprite = serializedObject.FindProperty("m_LightCookieSprite");
+            m_ShapeLightSprite = serializedObject.FindProperty("m_SpriteLightCookie");
```

Comment:
although, i'm not too worried, because this is experiment, but adding a new ref, will make shapeLightSprite now points to null, where before it points to the same thing.